docs/index.html

```diff
@@ -1247,50 +1246,115 @@ <h4>Workflow Instance</h4>
-                    "definition": [
-                      "urn:uuid:n1552885-cc91-4bb3-91f1-5466a0be084e"
-                    ],
-                    // This refers to a Workflow Instance
-                    "instance": [
-                      "urn:uuid:f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"
-                    ]
-                  },
-                  "holder":{
-                    "id":"did:web:sender.example",
-                    "type":"Organization",
-                    "location":{
-                       "type":"Place",
-                       "geo":{
-                          "type":"GeoCoordinates",
-                          "latitude":"68.7083",
-                          "longitude":"4.6377"
-                       },
-                       "address":{
-                          "type":"PostalAddress",
-                          "organizationName":"Ratke - Bergstrom",
-                          "streetAddress":"21851 Ima Heights",
-                          "addressLocality":"O'Connellborough",
-                          "addressRegion":"Missouri",
-                          "postalCode":"65587",
-                          "addressCountry":"Cyprus"
-                       }
-                    }
-                 }
-                }
-                </pre
-          >
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/traceability/v1"
+  ],
+  "id": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
+  "type":
+  [
+    "VerifiablePresentation",
+    "TraceablePresentation"
+  ],
+  "workflow": {
+    "definition": [
+      "urn:uuid:n1552885-cc91-4bb3-91f1-5466a0be084e"
+    ],
+    // This refers to a Workflow Instance
+    "instance": [
+      "urn:uuid:f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"
+    ]
+  },
+  "holder":{
+    "id":"did:web:sender.example",
+    "type":"Organization",
+    "location":{
+        "type":"Place",
+        "geo":{
+          "type":"GeoCoordinates",
+          "latitude":"68.7083",
+          "longitude":"4.6377"
+        },
+        "address":{
+          "type":"PostalAddress",
+          "organizationName":"Ratke - Bergstrom",
+          "streetAddress":"21851 Ima Heights",
+          "addressLocality":"O'Connellborough",
+          "addressRegion":"Missouri",
+          "postalCode":"65587",
+          "addressCountry":"Cyprus"
+        }
+    }
+  }
+}
+        </pre>
+        </section>
+        <section>
+          <h4>Workflow Replace</h4>
+          <p>
+A holder MAY use the <code>replace</code> property of a <code>TraceablePresentation</code> 
+to communciate to a verifier that a previous presentation should be replaced with a new presentation.
+          </p>
+          <p>
+Holders SHOULD ensure the previous presentation was accepted before presenting a <code>replace</code> presentation.
+          </p>
+          <p>
+Holders MUST present the same credential types that were present in the previous presentation and no new credentials.
```

Comment:
Its a "replace" operation, not a partial update... this makes reasoning easier... another operation could be defined in the future for partial updates, but I don't recommend discussing this.

Reply:
I think this should be worded in a way to include the case where only 1 credential needs to be replaced so only one credential is sent. 

The way it reads now is I must send every credential that was sent in the original presentation whether it needs replacement or not.

I think a better flow is for the replace operation to define a mapping that says replace credential id X with credential id Y, rather than replace presentation X with Y, because it enables the issuer to not have to keep track of previously sent presentations and keep the reasoning focused only on the credentials.